configure.ac

```
@@ -1609,6 +1611,10 @@ if test x$BUILD_READLINE = xyes; then
     AC_DEFINE(HAVE_READLINE, 1, [Enable CLI readline support.])
 fi
 
+PKG_CHECK_VAR(bashcompdir, [bash-completion], [completionsdir], ,
+  bashcompdir="${sysconfdir}/bash_completion.d")
+AC_SUBST(bashcompdir)
```

I think this should work as an alternative for `PKG_CHECK_VAR`. You can also add a comment saying that `PKG_CHECK_VAR` should be used but it's not available in some distributions.

```suggestion
bashcompdir="$(pkg-config --variable=completionsdir bash-completion)"
if test "x${bashcompdir}" = "x"; then
    bashcompdir="${sysconfdir}/bash_completion.d")
fi
```

---

Yes, thanks. This seems simpler than more macros. I will update based on that.